### PR TITLE
Docs redesign

### PR DESCRIPTION
**Pull request by FabberXD**
**CHECK IF IT WORKS**

Updated css
Changed fonts
Fixed bugs ("Ask for help" or discord button text fix)

### CODE REVIEW

**Comment by gdevillele** [comment]
thank you @FabberXD ! The code looks good, but I need to discuss the changes with the rest of the team before approving it.

File: lua/docs/content/page.tmpl [hunk 1]
```diff
@@ -7,6 +7,11 @@
 		<!-- CSS -->
   		<link rel="stylesheet" href="/style/css/style.css">
   		<link rel="stylesheet" href="/style/highlight/atom-one-dark.css">
+		
+		<!-- FONTS -->
+		<link rel="preconnect" href="https://fonts.googleapis.com">
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+		<link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;500;600;700&display=swap" rel="stylesheet">
```
Comment (gdevillele) [inline lua/docs/content/page.tmpl:14]: you can change the main font in a separate PR, as it will need more discussions with the team

File: lua/docs/content/style/css/style.css [hunk 2]
```diff
@@ -525,7 +525,7 @@ pre {
   color: #fff;
   font-family: m6x11, sans-serif;
   letter-spacing: 1.4px;
-  font-size: calc(12px + 0.8vw);
+  font-size: calc(9px + 0.8vw);
```
Comment (gdevillele) [inline lua/docs/content/style/css/style.css:528]: this change of text size can be in another PR with the font family change

**Review by gdevillele — COMMENTED** [review]
updated review

**Comment by aduermael** [comment]
Sorry but we've been working on the docs in parallel and this is unfortunately very outdated now. Closing, but thanks anyway!
